src/ontology/mappings/ordo_metadata.sssom.yml

```diff
@@ -0,0 +1,16 @@
+mapping_set_id: http://purl.obolibrary.org/obo/mondo/mondo-ingest/mapping/ordo.sssom.tsv
+mapping_set_description: >
+  The file `ordo.sssom.tsv` is generated from within the mondo-ingest repository (https://github.com/monarch-initiative/mondo-ingest),
+  by running the command `make mappings/ordo.sssom.tsv` from within `src/ontology`.
+license: https://creativecommons.org/licenses/by/4.0/
+creator_id: https://orcid.org/0000-0002-2906-7319
+curie_map:
+  ICD10: http://apps.who.int/classifications/icd10/browse/2010/en#/
+  MedDRA: http://purl.bioontology.org/ontology/MEDDRA/
+  MESH: https://meshb.nlm.nih.gov/record/ui?ui=
+  oboInOwl: http://www.geneontology.org/formats/oboInOwl#
+  OMIM: https://omim.org/entry/
+  ORDO: http://www.orpha.net/ORDO/
```

Comment:
I noticed that the output TSV has the shows values like `ORDO:Orphanet_124` rather than `ORDO:124`. I suppose that needs fixing.

Reply:
Oh yes the ORDO prefix should be

https://github.com/monarch-initiative/mondo/blob/master/src/ontology/metadata/mondo.sssom.config.yml#L18

Reply:
thx. fixed and updated